src/collect-install.rst

```diff
@@ -34,14 +34,21 @@ If you need a different version of Collect, you can download from the web and in
   
   You can also `download the ODK Collect APK <https://github.com/getodk/collect/releases/latest>`_ to your computer and load it on your device via `adb <https://developer.android.com/studio/command-line/adb.html>`_ or another tool like `AirDroid <https://www.howtogeek.com/105813/control-your-android-from-a-browser-with-airdroid/>`_.
```

Comment:
We should probably shouldn't recommend AirDroid. Apps don't always get good over time. Let people pick their poison.